docs/upgrading/upgrading-from-30.md

```diff
@@ -1,14 +1,230 @@
+
+## Export data
+Before getting into running any command to export the data make sure you
+have a backup of the database of the site you are about to migrate.
+
+Having said that, let's get the data exported by running the command below in the
+root directory of the site:
+
+    app/nut database:export --file=my_site_data_export.yml --users
+
+![Export data command](images/export-warning-command.JPG)
+
+This command is doing a couple of things:
+
+- Using the `--file` option, it creates the `my_site_data_export.yml` file in the
+  root directory with all the exported data.
+- Using the `--users` option will export the data of the users and their credentials.
+
+**Warning:** Since the users are being exported with their passwords and other private
+data, keep the export file in a safe place. Do not share or commit this file to Git.
+
+There are other handy options that can be used when exporting the data, such as:
+
+- `--contenttype=CONTENTTYPE`: Contenttype name to export records for (can be used multiple times).
+  Use this option if you want to have your data exported by Contenttype.
+  For example `php app/nut database:export --contenttype=news --file=news.yml` will
+  export all the records of the News Contenttype into the `news.yml` file
+- `--directory=DIRECTORY`: A destination directory. The command will automatically
+  generate file names. Use this option to export the data into a directory.
+  For example: `php app/nut database:export --contenttype=news --directory=export/news.yml`
+
+Those seem to be the options you would use the most but there exist other options
+you might find interesting. You can check them up by running:
+
+    app/nut database:export --help
+
+![Export data command](images/export-command-options.JPG)
+
+## Update to Bolt v5.0
+Even though you could directly update your existing site from Bolt 3 to Bolt 5, we will use
+a less complex approach which will also help to keep track of the old Bolt 3 installation
+and the new Bolt 5 version.
+
+### New Bolt 5 project install
+Let's get started by installing a new Bolt 5 project. You can create a new Bolt 5 project by
+running the command:
+
+    composer create-project bolt/project myprojectname
+
+`myprojectname` can be anything you like. This is just the name of the directory the installation
+will be put in.
+
+### Moving Bolt 3 files into Bolt 5 installation
+Once you have installed the latest version of Bolt 5 move forward by copying relevant
+files of yout Bolt 3 site into the recently installed version 5.
+In the table below you can see an overview of the list of directories and files to be copied and  
+the location in the new version's structure.
+
+| Bolt 3                                | Bolt 5                                      |
+| ------------------------------------- | -------------------------------------------- |
+| General config directory `app/config`               | `config/bolt`                  |
```

Comment:
Config endings has changed from `yml` to `yaml`.

Reply:
I'm not referring to any `.yml` file on the table. What do you mean?